Extension/lib/filter/rules/url-filter-rule.js

```diff
@@ -560,7 +560,11 @@
                     } else if (optionName[0] === api.FilterRule.NOT_MARK && optionName.substring(1) in UrlFilterRule.contentTypes) {
                         restrictedContentType |= UrlFilterRule.contentTypes[optionName.substring(1)]; // jshint ignore:line
                     } else if (optionName in UrlFilterRule.ignoreOptions) { // jshint ignore:line
-                        // Ignore
+                        if (optionName === 'CONTENT' && optionsParts.length === 1) {

```

Comment:
@atropnikov frankly, I agree with @Mizzick, and I like his implementation much more than what we have at the moment.

`$content` is a document-level exception modifier, just like `$elemhide`. There're no rules with `$content` and `$domain`  at the same time.